src/protectionmarketlist.schema.json

```diff
@@ -0,0 +1,447 @@
+        }
+      },
+      "required": ["major", "minor", "patch"]
+    },
+    "TagIdentifier": {
+      "type": "string",
+      "description": "The unique identifier of a tag",
+      "minLength": 1,
+      "maxLength": 30,
+      "pattern": "^[\\w ]+$",
+      "examples": ["yearn", "stablecoin", "money market", "protection market"]
+    },
+    "ExtensionIdentifier": {
+      "type": "string",
+      "description": "The name of a market extension property",
+      "minLength": 1,
+      "maxLength": 30,
+      "pattern": "^[\\w ]+$",
+      "examples": ["is_audited", "is_triggered"]
+    },
+    "ExtensionValue": {
+      "anyOf": [
+        {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 42,
+          "examples": ["#00000"]
+        },
+        {
+          "type": "boolean",
+          "examples": [true]
+        },
+        {
+          "type": "number",
+          "examples": [15]
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "TagDefinition": {
+      "type": "object",
+      "description": "Definition of a tag that can be associated with a market via its identifier",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the tag",
+          "pattern": "^[ \\w]+$",
+          "minLength": 1,
+          "maxLength": 20
+        },
+        "description": {
+          "type": "string",
+          "description": "A user-friendly description of the tag",
+          "pattern": "^[ \\w\\.,:]+$",
+          "minLength": 1,
+          "maxLength": 200
+        }
+      },
+      "required": ["name", "description"],
+      "examples": [
+        {
+          "name": "Stablecoin",
+          "description": "A market that takes stablecoin deposits"
+        }
+      ]
+    },
+    "DivestScript": {
+      "type": "object",
+      "description": "Parameters of the contract containing a `divest` method to exit an investment opportunity",
+      "properties": {
+        "address": {
+          "type": "string"
+        },
+        "signature": {
+          "enum": [
+            "function divest(uint256[] amounts, address recipient, uint256[] claimRewards)"
+          ],
+          "type": "string"
+        }
+      }
+    },
+    "InvestScript": {
+      "type": "object",
+      "description": "Parameters of the contract containing an `invest` method to exit an investment opportunity",
+
+      "properties": {
+        "address": {
+          "type": "string"
+        },
+        "signature": {
+          "enum": ["function invest(uint256[] amounts, uint256[] approve)"],
+          "type": "string"
+        }
+      }
+    },
+    "InvestmentOpportunity": {
+      "type": "object",
+      "description": "Properties required for detailing how a user can enter a protected investment after borrowing from the protection market",
+      "properties": {
+        "dataURI": {
```

Comment:
should this also have a `"format": "uri",` the same way `logoURI` does?

Reply:
Good catch, added the `"format": "uri"` along with an `"example" `